### DISCUSSION TITLE
Latest One-Stop-Shop Configuration - worse?

### DISCUSSION THREAD

**Discussion by 4q1** [bassamanator/Sovol-SV06-firmware#35] [Q&A]
I upgraded hardware and installed the latest One-Stop-Shop Sovol SV06 Klipper Configuration.  My prints got worse.  Looking through the conf files I see these two lines removed from the previous version:
> pressure_advance: 0.027
> [include mainsail.cfg]

While the square corner was added:
> square_corner_velocity: 8.0

There seems to be too much plastic on the corners so that my corners now look bulbous on my test cubes.  Also the top layer seems rougher (maybe a bit over extruded?)

Have other noticed this?  I wend back to my old hardware and did a cube test print and it looks much better.  (greater speed makes everything worse, but mostly the corners.)

Why is **mainsail.cfg** no longer an include?  Why was **pressure_advance** dropped?

**Comment by bassamanator** [comment]
Those changes are ancient, something like 50, 100 (?) commits ago. The whole configuration has changed significantly since that era, has only gotten better, and is now inline with Klipper standards.

Feel free to use the older version of the config, though to everyone else I recommend staying inline with the current changes on this repo.

I find this particularly fascinating:
>I **upgraded hardware** and installed the latest One-Stop-Shop Sovol SV06 Klipper Configuration. My **prints got worse**.

>I **wend back to my old hardware** and did a cube test print and it **looks much better**. 

It's obvious that you have issues with your physical setup. And `square_corner_velocity: 8.0` is pulled directly from Sovol's official settings. All these settings are of course easily adjustable by the user.

You need to tune your printer, and, or, print slower.

**Reply by 4q1** [reply]
I am sorry, I didn't mean to make you feel attacked. I appreciate your work on this. I'm just relaying what I've found.  the time difference between the two different version was a month at most.  I have the old version on the old pi hardware, and all I need to do is plugin the USB cable and everything prints almost flawlessly with the old configuration. No SV06 adjustments necessary.  Clearly it's not the printer hardware. I was just curious what the changes were for. I'm just trying to understand my problems so I can fix them. Perhaps I did something different with the PID settings or bed leveling, or paper test.  I though I did everything the same. The only difference I can find between these two config file settings are the lines I listed, and some macros getting moved.  ( FYI the readme file still hasbeeper.cfg listed under Directory Structure when it's moved to misc-macros.cfg)

**Reply by bassamanator** [reply]
A month, so roughly 50 commits. 

>The only difference I can find between these two config file settings are the lines I listed, and some macros getting moved.

If the changes are so few, then you wouldn't be getting the results you are, would you? Either (A) there are more changes than you believe, or (B) you've done something wrong. I was going to spar with you a bit here, but I'd rather just solve your problem.

Do the following:
- Read all the documentation.
- Use only 'known' hardware.
- Backup your good settings.
- Delete the configuration. 
- Download the new configuration, but don't PID tune, z_offset, etc.
- Copy everything from `#*# <---------------------- SAVE_CONFIG ---------------------->` onward and paste into new `printer.cfg`, inclusive.
- Adjust USB port in `printer.cfg`.
- Change to `square_corner_velocity: 5.0` which is the default (how it was before), and add `pressure_advance: 0.027`, even though there is 1:100,000,000+ chance that that value is accurate for your printer. Having said that, it's still perfectly alright to use that value if you're happy with your prints. Every user should tune their own [pressure advance](https://www.klipper3d.org/Pressure_Advance.html?h=pressu#pressure-advance), however.
- You are now inline with this repo, and have adding your personal settings on top, which is the state every user should be in.
- Reprint a 'known' model, preferably the same gcode if possible.
- Compare the prints, and report back your results.
- If the prints are the same, then great, transition to new hardware and keep inline with the repo via this method.

**Comment by bassamanator** [comment]
Update?
